WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/TotalLikesUseCase.kt

```diff
@@ -188,15 +189,12 @@ class TotalLikesUseCase @Inject constructor(
     }
 
     private fun onLinkClicked(params: LinkClickParams) {
-        analyticsTracker.track(STATS_LATEST_POST_SUMMARY_POST_ITEM_TAPPED)
+        analyticsTracker.track(STATS_INSIGHTS_TOTAL_LIKES_GUIDE_TAPPED)
```

Comment:
This is actually not the on the card but the latest post link on the card.  However, we can fix it later if required